wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Followers: add detail list view

Fixes #11128 

On the Insights Followers card, selecting `View more` will now show a full list of Followers.
- When `View more` is selected, the filter tab (and thus the data) will be the tab selected on the overview card.
- All Followers returned by the API are displayed (i.e. the UI does not limit the number of rows, but the API only returns 20 rows).
- Pull to refresh updates the data.
- If one tab has data and the other does not, the empty tab will display `No data yet`.

To test:

---
- On a site with more than 6 followers, go to Insights > Followers > View more.
- Verify:
  - The tab titles match the overview card.
  - The selected tab match the overview card.

![followers_details](https://user-images.githubusercontent.com/1816888/53458096-fdd4e400-39f1-11e9-829b-12cb92839e03.png)

---
- On a site where there are no Followers of one of the types (ex: the site has WP followers but no email followers), go to Insights > Followers > View more.
- On the detail view, select the no data tab.
- Verify the view is:

<img width="350" alt="followers_empty_tab" src="https://user-images.githubusercontent.com/1816888/53458124-193fef00-39f2-11e9-91f5-f7c63ae98b72.png">
